cmd/edit/machinepool/nodepool.go

```diff
@@ -152,6 +154,48 @@ func editNodePool(cmd *cobra.Command, nodePoolID string,
 		npBuilder.TuningConfigs(inputTuningConfig...)
 	}
 
+	if isKubeletConfigSet || interactive.Enabled() {
```

Comment:
We have work on-going to refactor this command into the new runner format. I have deliberately followed the existing approach to avoid doing a large refactor in this PR.